chapter00-basics.tex

```diff
@@ -237,7 +237,7 @@ \section{Polynomials}
   Let $f(x) ∈ℂ[x]$, then $M(f) ≤ \|f(x)\|_2$. 
 \end{theorem}
 \begin{proof}
-  This clearly holds for the zero polynomial. Thus, let $f(x)≠0$ and suppose the roots $z_i$ are ordered such that $|z_1|,\dots,|z_k| >1$ and $|z_{k+1}|,\dots,|z_n| ≤1$ such that $M(f) = |a_n ⋅ z_1 \cdots z_k|$. We define the polynomial
```

Comment:
We do not need to treat constant but nonzero polynomials outside of the proof. An empty product is, per definition, 1. So the proof as I wrote it is correct.